cmd/tiller/hooks.go

```diff
@@ -63,7 +79,7 @@ type simpleHead struct {
 //
 // To determine hook type, this looks for a YAML structure like this:
 //
-//  kind: SomeKind
+//  kindse_serv	: SomeKind
```

Comment:
Fixing that.

Also renaming `sortHooks` to `sortManifests` because that is driving me crazy.